src/demos/head.js

```diff
@@ -1,32 +1,58 @@
 (function() {
-  const root = window.location.pathname.split("demos").shift();
+  const CSS = ["demos/demos.css", "build/calcite-app.css", "vendor/@esri/calcite-components/calcite.css"];
+
+  const SCRIPTS = [
+    {
+      src: "build/calcite-app.esm.js",
+      type: "module"
+    },
+    {
+      src: "build/calcite-app.js",
+      noModule: true
+    },
+    {
+      src: "vendor/@esri/calcite-components/calcite.esm.js",
+      type: "module"
+    },
+    {
+      src: "vendor/@esri/calcite-components/calcite.js",
+      noModule: true
+    }
+  ];
+
+  if (location.port >= 3333 && location.port < 3340) {
```

Comment:
Where is this port range coming from? Is it guaranteed?

Reply:
This is what I've seen when starting the dev server. Do you have any ideas?

Reply:
We _could_ assume if there's any port at all that the dev server is running and enable page reloading.